TYPO3.Neos/Classes/TYPO3/Neos/Command/NodeCommandControllerPlugin.php

```diff
@@ -123,8 +123,14 @@ public function invokeSubCommand($controllerCommandName, ConsoleOutput $output,
     public function generateUriPathSegments($workspaceName, $dryRun)
     {
         $baseContext = $this->createContext($workspaceName, []);
-        $baseContextSiteNodes = $baseContext->getNode('/sites')->getChildNodes();
+        $baseContextSitesNode = $baseContext->getNode('/sites');
+        if (!$baseContextSitesNode) {
```

Comment:
@aertmann - Wouldn't it be better to verify by the interface?  `!(baseContextSitesNode instanceof NodeInterface)'